code/modules/reagents/reagent_containers/borghydro.dm

```diff
@@ -23,6 +23,9 @@
 /obj/item/weapon/reagent_containers/borghypo/crisis
 	reagent_ids = list("tricordrazine", "inaprovaline", "anti_toxin", "tramadol", "dexalin" ,"spaceacillin")
 
+/obj/item/weapon/reagent_containers/borghypo/lost
+	reagent_ids = list("tricordrazine", "bicaridine", "dexalin", "anti_toxin", "tramadol", "spaceacillin")

```

Comment:
No inaprov?

Reply:
Inaprov seemed weak to me when you have all the other chems but I can add it if desired.

Reply:
Inaprov's got some mildly useful little things it does, might be nice to have it.